runtime/failpoint.go

```diff
@@ -61,19 +50,6 @@ func (fp *Failpoint) Acquire() (interface{}, error) {
 
 // Release is called when the failpoint exists.
 func (fp *Failpoint) Release() {
-	fp.cmu.RLock()
-	ctx := fp.ctx
-	donec := fp.donec
-	fp.cmu.RUnlock()
-	if ctx != nil && !fp.released {
-		<-ctx.Done()
-		select {
-		case <-donec:
-		default:
-			close(donec)
-		}
-	}
-
 	fp.mu.RUnlock()
```

Comment:
Why do we lock mutex for failpoint execution time (between `Acquire` and `Release`)? I'm just curious as I don't see any case that this would matter.

Reply:
FYI. We don't expect the failpoint being updated concurrently by users when [enabling](https://github.com/etcd-io/gofail/blob/master/runtime/runtime.go#L78-L81) or [disabling](https://github.com/etcd-io/gofail/blob/d0d2a96a6ef08f6a2b190e1e063b1351865ab683/runtime/runtime.go#L108-L109) a failpoint.